File: Marcusoft.BDD.WhiteStepHelper.VB/TestConfig.vb
Imports System.Configuration

''' <summary>
''' Reads configuration properties from .config
''' </summary>
''' <remarks></remarks>
Public Class TestConfig
    Public Shared ReadOnly Property ApplicationUnderTestPath As String
        Get
            Return ConfigurationManager.AppSettings("ApplicationUnderTestPath")
        End Get
    End Property

    Public Shared ReadOnly Property MainWindowName As String
        Get
            Return ConfigurationManager.AppSettings("MainWindowName")
        End Get
    End Property
End Class

File: Marcusoft.BDD.WhiteStepHelper/TestConfig.cs
using System.Configuration;

namespace Marcusoft.BDD.WhiteStepHelper
{
    /// <summary>
    /// Reads configuration properties from .config
    /// </summary>
    /// <remarks></remarks>
    public static class TestConfig
    {
        public static string ApplicationUnderTestPath
        {
            get { return ConfigurationManager.AppSettings["ApplicationUnderTestPath"]; }
        }

        public static string MainWindowName
        {
            get { return ConfigurationManager.AppSettings["MainWindowName"]; }
        }
    }
}
